src/d3d12/d3d12_resource_pool_texture.hpp

```diff
@@ -60,6 +60,10 @@ namespace wr
 		void ReleaseTemporaryResources() final;
 
 		d3d12::TextureResource* GetTextureResource(TextureHandle handle) final;
+
+		// Load a texture from file.
```

Comment:
Not following the style guide. doxygen won't generate proper docs from this.

Reply:
This